api/nakadi-event-bus-api.yaml

```diff
@@ -798,6 +798,141 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /subscriptions/{subscription_id}/events:
+    get:
+      tags:
+        - stream-api
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        ---
+      parameters:
+        parameters:
+          - name: subscription_id
+            in: path
+            type: string
+            description: Id of subscription
+            required: true
+          - name: commit_timeout
+            in: query
+            description: |
+              The amount of time allowed between commits. If no commit is performed within this amount of time - the
+              connection will be closed from Nakadi side. Specified in seconds.
+            type: integer
+            format: int32
+            required: false
+            default: 60
+          - name: max_uncommitted_event_count
+            in: query
+            description: |
+              The amount of uncommitted events Nakadi will stream before pausing the stream. When in paused state and
+              commit comes - the stream will continue.
```

Comment:
I think this could be interpreted in a slightly inaccurate way. It's not that stream will resume after a commit. But it would resume if the commit offset decreased the number of uncommitted events.